tests/testsuite/cargo_tree/deps.rs

```diff
@@ -1158,7 +1182,7 @@ fn format() {
             [dependencies]
             dep = {version="1.0", optional=true}
             other-dep = {version="1.0", optional=true}
-            dep_that_is_awesome = {version="1.0", optional=true}
+            dep_that_is_awesome = {version=">=1.0, <2", optional=true}
```

Comment:
please make test edits in their own commit. Even adding a place holder for the assert, just not yet using `{c}` yet. This commit would add that format option and the diff would be focused on just that and how the output changed.

Reply:
No problem, I'll split into commits.